drivers/hmis/app/graphql/mutations/ac_hmis/void_referral_request.rb

```diff
@@ -0,0 +1,51 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class AcHmis::VoidReferralRequest < CleanBaseMutation
+    description 'Void a referral request'
+
+    argument :referral_request_id, ID, required: true
+
+    field :record, Types::HmisSchema::ReferralRequest, null: true
+
+    def resolve(referral_request_id:)
+      return error_response('connection not configured') unless HmisExternalApis::AcHmis::LinkApi.enabled?
+
+      request = HmisExternalApis::AcHmis::ReferralRequest.active
+        .viewable_by(current_user)
+        .find_by(id: referral_request_id)
+      return error_response('referral request record not found') unless request
+
+      project = Hmis::Hud::Project.viewable_by(current_user).find(request.project_id)
+      return error_response('project record not found') unless project
+
+      allowed = current_user.permissions_for?(project, [:can_manage_incoming_referrals])
```

Comment:
Whoops I missed this. It's not working correctly, should be:
```suggestion
      allowed = current_user.can_manage_incoming_referrals_for?(project)
```

Reply:
Fixed in https://github.com/greenriver/hmis-warehouse/pull/3061

Reply:
Thanks! Yeah this was a bit of a guess taken from elsewhere in the code base; I should have called that out. None of the permissions work for me so I just hack around them, I should probably just toss my db and start from scratch.